hw/battery/src/battery.c

```diff
@@ -487,10 +487,15 @@ battery_set_poll_rate_ms(struct os_dev *battery, uint32_t poll_rate)
 {
     struct battery *bat = (struct battery *)battery;
 
-    if ((poll_rate == 0) || (bat == NULL)) {
+    if ((bat == NULL)) {
```

Comment:
Just a nitpick, but you could remove the extra set of parentheses here.

Reply:
of course!